app/models/user.js

```diff
@@ -26,6 +26,10 @@ const userSchema = new Schema({
     type: Boolean,
     default: false
   },
+  moderator: {
```

Comment:
This should instead be an `enum`, so we don't end up with a ton of bools. But this works for now I guess

Reply:
Yehh, I looked into it now. Creating the enum would be easy, but a lot of code use `req.user.admin` and `req.user.moderator` for permissions, so that will be a lot to rewrite.

Reply:
Aha, 

We can just keep it like this for now then. :+1: